client/network/src/protocol/sync.rs

```diff
@@ -1145,10 +1145,6 @@ impl<B: BlockT> ChainSync<B> {
 					if let Some(peer) = who.and_then(|p| self.peers.get_mut(&p)) {
 						peer.update_common_number(number);
 					}
-
```

Comment:
Why did you remove this?

Reply:
It's probably removed during the merge. There is now no diff with Substrate